src/ansys/pyensight/core/exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py

```diff
@@ -356,7 +371,9 @@ def help(self) -> None:
         self.warning("  --/exts/ansys.tools.omniverse.core/help=1")
         self.warning("     Display this help.")
         self.warning("  --/exts/ansys.tools.omniverse.core/run=1")
-        self.warning("     Run the server.")
+        self.warning("     Run the server until the DSG connection is broken.")
+        self.warning("  --/exts/ansys.tools.omniverse.core/run=0")

```

Comment:
Include "one-shot" mode that does a single DSG pull and conversation.  Useful for many things (e.g. "save as USD").